sway/commands/titlebar_border_thickness.c

```diff
@@ -21,8 +21,11 @@ struct cmd_results *cmd_titlebar_border_thickness(int argc, char **argv) {
 
 	for (int i = 0; i < root->outputs->length; ++i) {
 		struct sway_output *output = root->outputs->items[i];
-		arrange_workspace(output_get_active_workspace(output));
-		output_damage_whole(output);
+		struct sway_workspace *ws = output_get_active_workspace(output);
+		if (ws) {
```

Comment:
Can this be executed outside of a command handler?

Reply:
Nope. Just missed it